packages/devtools_app/lib/src/screens/memory/memory_graph_model.dart

```diff
@@ -249,7 +253,7 @@ class HeapGraph {
 
   /// Group all classes by libraries (key is library, value are classes).
   /// This is the entire set of objects (no filter applied).
-  final Map<String, Set<HeapGraphClassLive>> rawGroupByLibrary = {};
+  final Map<String, Set<HeapGraphClassLive?>> rawGroupByLibrary = {};
```

Comment:
why does HeapGraphClassLive need to be nullable?

Reply:
fixed